### PR TITLE
Fixed object relation select values

### LINKED ISSUE #7132: [DataObject] Missing admin translations on object relations
# Bug Report

### Expected behavior
Excepted behvaiour

That the ~column-names and~ select/multiselect values are translated as in the object editing.

### Actual behavior
Actual behvaiour

My bad: The column title is translated correctly, i just forgot the translation in the admin translation.

### Steps to reproduce
Steps to repoduce 1

Steps to repoduce 2

### PR DESCRIPTION
**Pull request by skochTC**
Closes #7132

### CODE REVIEW

File: bundles/AdminBundle/Resources/public/js/pimcore/object/tags/advancedManyToManyObjectRelation.js [hunk 1]
```diff
@@ -131,9 +131,15 @@ pimcore.object.tags.advancedManyToManyObjectRelation = Class.create(pimcore.obje
                 fc.layout = field;
                 fc.editor = null;
                 fc.sortable = false;
-                if(fc.layout.key === "fullpath") {
+
+                if (fc.layout.key === "fullpath") {
                     fc.renderer = this.fullPathRenderCheck.bind(this);
+                } else if(fc.layout.layout.fieldtype == "select" || fc.layout.layout.fieldtype == "multiselect") {
```
Comment (dvesh3) [inline bundles/AdminBundle/Resources/public/js/pimcore/object/tags/advancedManyToManyObjectRelation.js:137]: please fix it for grid view as well..thanks! 
image

somewhere here: https://github.com/pimcore/pimcore/blob/e6f3de97c326595166907595a56e33176aa2cc11/bundles/AdminBundle/Resources/public/js/pimcore/object/helpers/grid.js#L406

Comment (dvesh3) [reply bundles/AdminBundle/Resources/public/js/pimcore/object/tags/advancedManyToManyObjectRelation.js:137]: ping @skochTC

Comment (skochTC) [reply bundles/AdminBundle/Resources/public/js/pimcore/object/tags/advancedManyToManyObjectRelation.js:137]: Yeah, sure. We will adapt this to the grid view.

But we have a lot to do this week.

Comment (tvedderTC) [reply bundles/AdminBundle/Resources/public/js/pimcore/object/tags/advancedManyToManyObjectRelation.js:137]: I've changed this, it's now also working for a grid-view.

File: bundles/AdminBundle/Resources/public/js/pimcore/object/helpers/grid.js [hunk 2]
```diff
@@ -300,6 +300,12 @@ pimcore.object.helpers.grid = Class.create({
 
                         fc.locked = this.getColumnLock(field);
 
+                        if ((fields[i].layout.fieldtype == "select" || fields[i].layout.fieldtype == "multiselect") && fields[i].layout.options.length > 0) {
+                            fields[i].layout.options.forEach(option => {
```
Comment (dvesh3) [inline bundles/AdminBundle/Resources/public/js/pimcore/object/helpers/grid.js:304]:
```suggestion
                        if ((fieldType === "select" || fieldType === "multiselect") && field.layout.options.length > 0) {
                           field.layout.options.forEach(option => {
```

File: bundles/AdminBundle/Resources/public/js/pimcore/object/helpers/grid.js [hunk 3]
```diff
@@ -439,6 +445,22 @@ pimcore.object.helpers.grid = Class.create({
                         let colName = columnKeys[col];
                         result += '<td style="padding: 0 5px 0 5px; font-size:11px; border-bottom: 1px solid #d0d0d0;  border-top: 1px solid #d0d0d0; border-left: 1px solid #d0d0d0; border-right: 1px solid #d0d0d0;">';
                         let displayValue = item[colName] ? item[colName] : "&nbsp";
+                        let colType = field.layout.columns[col].type;
+                        let colValues = field.layout.columns[col].value;
+
+                        // Replace multiple values
+                        if (colType == "select" || colType == "multiselect") {
```
Comment (dvesh3) [inline bundles/AdminBundle/Resources/public/js/pimcore/object/helpers/grid.js:452]:
```suggestion
                        if (colType === "select" || colType === "multiselect") {
```

Comment (tvedderTC) [reply bundles/AdminBundle/Resources/public/js/pimcore/object/helpers/grid.js]: Changed.

File: bundles/AdminBundle/Resources/public/js/pimcore/object/helpers/grid.js (hunk 2 above)
Comment (tvedderTC) [reply bundles/AdminBundle/Resources/public/js/pimcore/object/helpers/grid.js]: Changed.

**Review by dvesh3 — CHANGES_REQUESTED** [review]
meta column value is duplicated on Grid:
before changes:
image

After change:
image

**Comment by brusch** [comment]
@skochTC ping 🏓😊

**Comment by brusch** [comment]
@dvesh3 can you please re-review? Thanks a lot! 😊

**Comment by dvesh3** [comment]
@skochTC looks good ..one last thing, can you please rebase the PR on 6.8 branch, since it is fix. thanks a lot!

**Comment by skochTC** [comment]
> @skochTC looks good ..one last thing, can you please rebase the PR on 6.8 branch, since it is fix. thanks a lot!

I am very busy 😕 @tvedderTC has done that 🙂

**Comment by brusch** [comment]
Thanks a lot @skochTC @tvedderTC !

**Pull request merged**